### ISSUE TITLE
Display tooltip on the custom emoji

### ISSUE THREAD

**Issue by anosidium** [divadretlaw/EmojiText#24]
EmojiText: 3.6.0
macOS: 14.3 (23D56)

Once again, thank you for fixing the custom emoji text height.

Please, I would like to have tooltip displayed on the custom emoji. If you browse the website and place the cursor on the custom emoji, it will display a tooltip showing the custom emoji shortcode. I was thinking by default, it should do that but allow the developer to customise it to display anything else.

The API could look something like this for `RemoteEmoji`:
```Swift
RemoteEmoji(shortcode: ..., url: ..., help: ":customshortcode:")
```

I called it `help` because that is what it is called in SwiftUI whereas in Cocoa and Cocoa Touch, it is called `tooltip`, sometimes `toolTip`.

**Comment by divadretlaw** [comment]
Agreed, that would be a nice addition. I was looking into that not too long ago, but I haven't found a proper way to add that.

If you find a way to add `help` / `tooltip` to an interpolated `Image` in `Text` let me know.

**Comment by divadretlaw** [comment]
Could possibly be implemented with the new `EmojiTextView`

See https://github.com/divadretlaw/EmojiText/issues/30#issuecomment-3193727144
